gridstatus/pjm.py

```diff
@@ -80,7 +80,330 @@ class PJM(ISOBase):
         "124076095",
         "970242670",
         "1709725933",
-    ]
```

Comment:
are these zones? if not, perhaps we create a new variable called `selected_nodes`? i like this approach to because it's is backwards compatible too